cmd/cluster/create.go

```diff
@@ -257,8 +257,13 @@ func CreateCluster(ctx context.Context, opts Options) error {
 			fmt.Println("---")
 		}
 	default:
+		lbl := "hypershift-cli-" + infra.InfraID
 		for _, object := range exampleObjects {
 			key := crclient.ObjectKeyFromObject(object)
+			if object.GetObjectKind().GroupVersionKind().Kind == "Secret" {
```

Comment:
I wonder if we even need to filter — why not apply it to everything the CLI creates?